### PR TITLE
PP-8162: Deploy carbon-relay to production

### PR DESCRIPTION
**Pull request by oswaldquek**
[Succesful deploy to prod](https://cd.gds-reliability.engineering/teams/pay-deploy/pipelines/deploy-to-production/jobs/deploy-carbon-relay-to-prod/builds/2).

### CODE REVIEW

File: ci/pipelines/deploy-to-production.yml [hunk 1]
```diff
@@ -346,6 +356,60 @@ groups:
...
+          AWS_REGION: "eu-west-1"
+          CLUSTER_NAME: "production-2-fargate"
+          APP_NAME: carbon-relay
+          CARBON_RELAY_IMAGE_TAG: ((.:carbon_relay_image_tag))
+      - task: deploy-to-prod
+        file: pay-ci/ci/tasks/deploy-carbon-relay.yml
+        params:
+          <<: *aws_assumed_role_creds
+          CARBON_RELAY_IMAGE_TAG: ((.:carbon_relay_image_tag))
+          STUNNEL_IMAGE_TAG: test
```
Comment (katstevens) [inline ci/pipelines/deploy-to-production.yml:400]: So this would be `0-release` at the moment for prod

Comment (oswaldquek) [reply ci/pipelines/deploy-to-production.yml]: I noticed I hadn't committed it at the same time you commented! Pushed now.

File: ci/pipelines/deploy-to-production.yml [hunk 2]
```diff
@@ -346,6 +356,60 @@ groups:
       - update-deploy-to-prod-pipeline
 
 jobs:
+  - name: deploy-carbon-relay-to-prod
```
Comment (katstevens) [inline ci/pipelines/deploy-to-production.yml:359]: I think positioning it first in the `jobs` list affects the ordering of the tabs in the pipeline - we should prob keep it consistent with staging and test and stick it between toolbox and the sidecars.

Comment (oswaldquek) [reply ci/pipelines/deploy-to-production.yml]: Fixed

**Review by katstevens — APPROVED** [review]

**Pull request merged**